### PR TITLE
Japanese translation

### PR DESCRIPTION
**Pull request by eiida**
Hi. Please put "ja-deeplearning-model-import-keras" online. The translation is ready. 
Hi. Please put "ja-deeplearning-mnist-for-beginners" online. The translation is ready.
Please put "deeplearning-compare-dl4j-torch7-pylearn" online. The translation is ready.
Please put "visualization" online. The translation is ready.
Please update the translation of "accuracy". It was revised today.

## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

Please review
https://github.com/deeplearning4j/deeplearning4j/blob/master/CONTRIBUTING.md before opening a pull request.

### CODE REVIEW

File: ja/visualization.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+これは比較的簡単です。
+
+```
+    //ユーザーインターフェースのバックエンドを初期化する
+    UIServer uiServer = UIServer.getInstance();
+
+    //ネットワーク情報（勾配、スコア対時間など）が保管される場所の情報を設定するこちらのメモリーに保管します。
+    StatsStorage statsStorage = new InMemoryStatsStorage();         //その他の選択肢： 新しいFileStatsStorage(File)。これは後で保存、ロードするため。
+    
+    //Attach the StatsStorageインスタンスをUIに付ける。これによりStatsStorageのコンテンツが視覚化される。
```
Comment (saudet) [inline ja/visualization.md]: @eiida 「Attach the」が間違って残っています。

File: ja/deeplearning-compare-dl4j-torch7-pylearn.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+また、JavaはScala、Clojure、Python、Rubyなど他の人気のある言語からネイティブで使用することができます。Javaを選ぶことにより、我々はできるだけ最も少ない主要なプログラミングのコミュニティーを除外しました。 
+
+JavaはCやC++ほど素早くはありませんが、多くの人が思い込んでいるよりも速度は速いため、我々はGPUであろうとCPUであろうとノードを少し追加すると加速できるよう、分散システムを構築しました。つまり、速度を向上させたければ、より多くの箱を投げつければいいのです。 
+
+最後に我々はJavaにDL4J用のND-Arrayを含めて、Numpyの基本的アプリケーションを構築しています。我々はJavaの欠点の多くは素早く解決でき、その長所の多くはしばらくの間なくなることはないはずだと見ています。 
+
+### <a name="scala">Scala</a>
+
+我々はDeeplearning4jとND4Jを構築するにあたって[Scala](./scala)に特に注目しましました。データサイエンスにおいてScalaが優勢な言語となる可能性があると見ているからです。数値計算を書くこと、ベクトル化、[Scala API](http://nd4j.org/scala.html)の実装されたJVM用のディープラーニングライブラリによってコミュニティーはその目標に向かうことができるのです。
```
Comment (eiida) [inline ja/deeplearning-compare-dl4j-torch7-pylearn.md]: Typo in where?

File: ja/mnist-for-beginners.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+            .iterations(1)
+            .learningRate(0.006)
+            .updater(Updater.NESTEROVS).momentum(0.9)
+            .regularization(true).l2(1e-4)
+            .list()
+  </code></pre>
+  <h5>.seed(rngSeed)</h5>
+  <p>このパラメータはある特定のランダムに生成された重み初期化を使用します。あるサンプルを何回も実行し、毎回開始時に新しい重みをランダムに生成すると、結果（F1値や正答率）にかなりの違いが生じるかもしれません。というのは、初期の重みが異なるとアルゴリズムでエラースケープの極小値が異なってしまうかもしれないからです。重みを同じランダムなものに保っておくと、他の条件を平等に保ったまま、他のハイパーパラメタ―をより正確に調整することによる効果のみに限って確認することができます。</p>
+  <h5>.optimizationAlgo(OptimizationAlgorithm.STOCHASTIC_GRADIENT_DESCENT)</h4>
+  <p>確率的勾配降下法（Stochastic gradient descent；SGD）は、コスト関数を最適化するのに一般に使用される方法です。エラーを最小限に抑える確率的勾配降下法や他の最適化アルゴリズムについて知るには、<a href="https://www.coursera.org/learn/machine-learning" target="_blank">Andrew Ngfs Machine Learning course</a>、弊社の<a href="http://deeplearning4j.org/glossary#stochasticgradientdescent" target="_blank">グロッサリー</a>にある確率的勾配降下法の定義をご参照ください。</p>
```
Comment (eiida) [inline ja/mnist-for-beginners.md]: Correction made.

File: ja/mnist-for-beginners.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+  <h5>.iterations(1)</h5>
+  <p>ニューラルネットワークのイテレーションを行うごとに、学習を一歩進めたことになります。つまり、モデルの重み更新を一回行ったことになるのです。ネットワークはデータを目の当たりにし、そのデータについて推論し、その推論がどのくらい間違っていたかに基づいて自身のパラメータを修正します。従って、イテレーションを多く行えば行うほど、ニューラルネットワークはより多く進歩し、学習し、エラーを最小限に抑えることができるのです。</p>
+  <h5>.learningRate(0.006)</h5>
+  <p>このコマンドで学習率を指定します。学習率とはイテレーション一回につき重みに対して行われた調節のサイズ、つまりステップサイズにあたります。学習率が高いとネットはerrorscapeを素早く巡回しますが、最小エラーのレベルを超えてしまう傾向があります。学習速度が低いと、最小値を見つける可能性は高まりますが、非常に遅い速度で行われます。重み調節が小さいステップで行われるためです。</p>
+  <h5>.updater(Updater.NESTEROVS).momentum(0.9)</h5>
+  <p>慣性項（momentum）は最適なポイントに最適化アルゴリズムがどれだけ素早く収束するかを決定する要素のうち一つです。慣性項は、重みが調節される方向に影響するため、コーディングの世界では、一種の重みの<code>アップデーター</code>と見なします。</p>
+  <h5>.regularization(true).l2(1e-4)</h5>
+  <p>正規化（regularization）とは、<b>過剰適合（overfitting）</b>を回避するためのテクニックです。過剰適合とは、あるモデルがトレーニングのデータには非常によく適合しても、実際に使用された時に過去に接したことのないデータに出くわすやいなや非常にパフォーマンスが悪くなることを言います。</p>
```
Comment (eiida) [inline ja/mnist-for-beginners.md]: Typo in where?

File: ja/mnist-for-beginners.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+  <p>弊社では、L2正規化を使用することにより、個々の重みが全体の結果に大きな影響を及ぼさないよう回避しています。</p>
+  <h5>.list()</h5>
+  <p>ネットワーク内の層数を指定します。この関数は、設定をn回複製し、層に基づいた設定を構築します。</p>
+  <p>上記の説明で分かりにくいことがあれば、先に挙げた<a href="https://www.coursera.org/learn/machine-learning" target="_blank">Andrew Ngfs Machine Learning course</a>をご参照になることをお勧めします。</p>
```
Comment (eiida) [inline ja/mnist-for-beginners.md]: Correction made.

File: ja/mnist-for-beginners.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+    <li>規模を広げて、Spark、Hadoop、Kafkaなどの主要なJVMフレームワークと統合させることができる。</li>
+    <li>分散CPU及び/または分散GPUで実行に最適化されている。</li>
+    <li>JavaやScalaのコミュニティに貢献している。</li>
+    <li>導入された企業様への商業用サポートを行っている。</li>
+  </ul>
+  <p>ご質問のある方は弊社のオンライン<a href="https://gitter.im/deeplearning4j/deeplearning4j" target="_blank">Gitter サポートチャットルーム</a>にてご連絡ください。</p>
+  <ul class="categorized-view view-col-3">
+    <li>
+      <h5>その他のDeeplearning4jのチュートリアル</h5>
+      <a href="http://deeplearning4j.org/neuralnet-overview">Introduction to Neural Networks</a>
```
Comment (eiida) [inline ja/mnist-for-beginners.md]: Some links are not in Japanese but all other links available in Japanese are updated from English ones.

**Comment by eiida** [comment]
Please put the Japanese translation "deeplearning-compare-dl4j-torch7-pylearn.md" online. Thanks.

**Comment by eiida** [comment]
Please put the Japanese translation of "deeplearning-model-import-keras" online.
https://github.com/eiida/deeplearning4j/blob/gh-pages/ja/deeplearning-model-import-keras.md

**Comment by eiida** [comment]
Please put the Japanese version https://github.com/eiida/deeplearning4j/blob/gh-pages/ja/mnist-for-beginners.md online.

**Comment by eiida** [comment]
Please add https://github.com/eiida/deeplearning4j/blob/gh-pages/ja/visualization.md online.

**Comment by eiida** [comment]
Please add https://github.com/eiida/deeplearning4j/blob/gh-pages/ja/troubleshootingneuralnets.md online.

**Comment by eiida** [comment]
Please add the Japanese page https://github.com/eiida/deeplearning4j/blob/gh-pages/ja/accuracy.md online.

**Comment by agibsonccc** [comment]
Hi,

I will wait on @saudet to merge this. Thanks!

File: ja/deeplearning-compare-dl4j-torch7-pylearn.md [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+* (-) 商業用サポート無し
+* (-) 新しいトレーニングのバッチがあるごとに読み込みをするためにPythonにドロップアウトする。
+* (-) ツールにはなりにくい。
+* （ - ）大規模なソフトウェアプロジェクトで動的型付けにエラーが発生しやくすい。
+
+### <a name="caffe">Caffe</a>
+
+[**Caffe**](http://caffe.berkeleyvision.org/)は有名で、かつ広く一般に使用されているマシンビジョンのライブラリで、素早い畳み込みネットワークのMatlabによる実装をC及びC++に移植したものです（速度とこのテクニカル面での負担とのトレードオフを考慮したい方は、あるチップから別のチップにC++を移植することについてのteve Yegge氏の見解を[こちら](https://sites.google.com/site/steveyegge2/google-at-delphi)でお読みいただけます）。Caffeは、テキスト、音声、時系列データなど他のディープラーニングのアプリケーションを対象としたものではありません。これまでご紹介してまいりました他のフレームワークと同様に、CaffeはそのAPIが目的でPythonを選択しました。 
+
+Deeplearning4jとCaffeはどちらも畳み込みネットワークを使用して、最先端の技術で画像分類を行います。Caffeとは対照的に、Deeplearning4jはGPUの並列処理*サポート*を提供しているため、任意数のチップに使用できるだけでなく、数多くの特徴に対応できるため、ディープラーニングがよりスムースに複数のGPUクラスタを使って並列処理できます。論文でもよく引用されているように、CaffeはそのModel Zooサイトがホストする事前トレーニングされたモデルのソースとして主に使用されます。Deeplearning4jは、CaffeモデルをSparkにインポートするために[構文解析ツールを構築しています](https://github.com/deeplearning4j/deeplearning4j/pull/480)。
```
Comment (saudet) [inline ja/deeplearning-compare-dl4j-torch7-pylearn.md]: @eiida URLが間違っているのではなく翻訳は英語版とは一致していないと言いたかったのです。

**Comment by saudet** [comment]
It doesn't reflect some recent bits changed w.r.t Caffe and Caffe2 at https://deeplearning4j.org/compare-dl4j-torch7-pylearn#caffe but everything else looks good to me. Merge?

**Comment by eiida** [comment]
Please put the following Japanese page online:
https://github.com/eiida/deeplearning4j/edit/gh-pages/ja/buildinglocally.md
